src/main/webapp/dataverse_header.xhtml

```diff
@@ -31,21 +39,35 @@
                 </div>
                 <div class="collapse navbar-collapse" id="topNavBar">
                     <ul class="nav navbar-nav navbar-right">
-                        <li class="dropdown">
-                            <a href="#" class="dropdown-toggle" data-toggle="dropdown">#{bundle['dataverse.results.btn.addData']} <b class="caret"></b></a>
-                            <ul class="dropdown-menu">
-                                <li>
-                                    <h:outputLink value="/dataverse.xhtml?ownerId=#{dataverseServiceBean.findRootDataverse().id}">
-                                        <h:outputText value="#{bundle['dataverse.results.btn.addData.newDataverse']}"/>
-                                    </h:outputLink>
-                                </li>
-                                <li>
-                                    <h:outputLink value="/dataset.xhtml?ownerId=#{dataverseServiceBean.findRootDataverse().id}">
-                                        <h:outputText value="#{bundle['dataverse.results.btn.addData.newDataset']}"/>
-                                    </h:outputLink>
+                        <ui:fragment rendered="#{showAddDataGroup}">
+                        <ui:fragment rendered="#{!authUserSession}">
+                            <li>
+                                <a href="#" onClick="PF('addData_popup').show()">#{bundle['dataverse.results.btn.addData']} </a>
```

Comment:
This `addData_popup` reference that was copied over from search-include-fragment will now look for that dialog on every page, but the dataverse pg is the only page that will have the dialog included on it.

Reply:
I made the add data popup into a fragment and included it in the dataverse template.